bot/feedback/feedback.py

```diff
@@ -0,0 +1,303 @@
+                if constants.LIMIT_SUGGESTION_VOTES < reactions[0].count > reactions[1].count and \
+                        difference > (total_votes / 2):
+                    new_embed = _recolor_embed(message.embeds[0], constants.EMBED_COLOR_SUGGESTION_MEMBERS_LIKE)
+                    await message.edit(embed=new_embed)
+                elif constants.LIMIT_SUGGESTION_VOTES < reactions[1].count and difference > (total_votes / 2):
+                    new_embed = _recolor_embed(message.embeds[0], constants.EMBED_COLOR_SUGGESTION_MEMBERS_DISLIKE)
+                    await message.edit(embed=new_embed)
+
+
+async def _notify_user_suggestion_change(guild: discord.Guild, user_id: int):
+    """Method which notifies a user about changes regarding his suggestion.
+
+    Gets the corresponding Discord member and sends a personalized DM to him informing him about changes regarding his
+    suggestion.
+
+    Args:
+        guild (discord.Guild): The Discord server on which to find the suggestion.
+        user_id (int): The id of the user who submitted the suggestion.
+    """
+    member = guild.get_member(user_id)
+    name = member.nick if member.nick else member.name
+    text = "Hey, {0}!\nEs gibt Neuigkeiten bezüglich deines Vorschlags.Sieh am besten gleich in <#{1}> nach, wie " \
+           "das Urteil ausgefallen ist. :fingers_crossed:".format(name, constants.CHANNEL_ID_SUGGESTIONS)
+
+    await member.send(text)
+
+
+async def _refresh_suggestion_embed(message: discord.Message, author: discord.Member, reason: Optional[str],
+                                    status: SuggestionStatus):
+    """Method which adapts an embed regarding a suggestion.
+
+    Changes title, color and fields of the embed accordingly to the new status of the suggestion.
+
+    Args:
+        message (discord.Message): The message containing the suggestion embed.
+        author (discord.Member): The user who invoked the command to change it.
+        reason (Optional[str]): The reason for the decision.
+        status (SuggestionStatus): The new status of the suggestion.
+    """
+    dict_embed = message.embeds[0].to_dict()
+
+    if "fields" in dict_embed:
+        dict_embed["fields"].clear()
+    else:
+        dict_embed["fields"] = list()
+
+    if reason:
+        dict_embed["fields"].append({"name": f"Begründung von {author}:", "value": reason, 'inline': False})
+    else:
+        dict_embed = message.embeds[0].to_dict()
+
+    dict_embed["title"] = dict_embed["title"].split(" -")[0]
+    if status == SuggestionStatus.APPROVED:
+        dict_embed["color"] = constants.EMBED_COLOR_SUGGESTION_APPROVED
+        dict_embed["title"] = dict_embed["title"] + " - Genehmigt :white_check_mark:"
+    elif status == SuggestionStatus.DENIED:
+        dict_embed["color"] = constants.EMBED_COLOR_SUGGESTION_DENIED
+        dict_embed["title"] = dict_embed["title"] + " - Abgelehnt :no_entry_sign:"
+    elif status == SuggestionStatus.CONSIDERED:
```

Comment:
Could you possibly assign the color and Text to the enum (for example as a tuple) and just write:

```
# suggestion_status.py
class SuggestionStatus(enum):
    ...
    DENIED = (2, constants.EMBED_COLOR_SUGGESTION_DENIED, " - Abgelehnt :no_entry_sign:")
    ....

#feedback.py (the commented code)
dict_embed["color"] = status[1]
dict_embed["title"] = dict_embed["title"] + status[2]
```
No need for if/else then

Reply:
I could do that but then it would break the code where I get a value from the DB and try to get an enum constant with it. For this to work I would either have to use the complete tuple to get the constant or write some method which returns the constant by providing the first value of the tuple. Either way I don't think this is needed just because some if/elif.